src/datasets/utils/tf_utils.py

```diff
@@ -173,6 +174,21 @@ def dataset_to_tf(
     else:
         raise ImportError("Called a Tensorflow-specific function but Tensorflow is not installed.")
 
+    # TODO Matt: When our minimum Python version is 3.8 or higher, we can delete all of this and move everything
```

Comment:
Hi Matt, is `datasets` going to drop Python 3.7 support due to its upcoming EOL? Because it will happen by the end of the month in case we want to wait and set the minimum version to 3.8, even though I assume some users may still be using 3.7?